libsql/src/sync.rs

```diff
@@ -18,7 +18,27 @@ impl SyncContext {
         }
     }
 
-    pub(crate) async fn push_with_retry(
+    pub(crate) async fn push_one_frame(
```

Comment:
Is this still guarded with `#[cfg(feature = "sync")]` like previously?

Reply:
Not needed since `sync.rs` is behind that feature flag.